meinberlin/apps/ideas/forms.py

```diff
@@ -1,15 +1,32 @@
 from django import forms
+from django.utils.translation import ugettext_lazy as _
 
 from adhocracy4.categories.forms import CategorizableFieldMixin
 
 from . import models
 
 
 class IdeaForm(CategorizableFieldMixin, forms.ModelForm):
+    right_of_use = forms.BooleanField(required=False)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if self.instance.name != '' and self.instance.image:
```

Comment:
I think you can check for `self.instance.pk` here, as the instance won't have a pk when it has not been saved

Reply:
Yes, thanks! But even just checking for the image should be enough!